### PR TITLE
fix(toggles): a11y - convert toggles to radio buttons - Fixes #1623

### LINKED ISSUE #1623: a11y: Refactor chart toggles
The chart toggles at `/src/components/common/toggle` need an accessibility refactor:

1. Change the `spans` to buttons
2. Define the toggle as an aria group

### PR DESCRIPTION
**Pull request by LaurieSReynolds**
Convert toggle to use radio button to support a11y navigation.

image

---

radio-toggle

### CODE REVIEW

File: src/__tests__/components/common/radio-toggle.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import RadioToggle from '~components/common/radio-toggle'
+
+describe('Components : Common: Card', () => {
```
Comment (kevee) [inline src/__tests__/components/common/radio-toggle.js:5]: Let's rename to `Components : Common : Radio toggle`

**Review by kevee — APPROVED** [review]
This looks perfect, @LaurieSReynolds! I just left one little change to update in the test description and then I think we're good to go.

**Review by kevee — CHANGES_REQUESTED** [review]
Whoops, meant to request change in https://github.com/COVID19Tracking/website/pull/1662#discussion_r539441139

File: src/__tests__/components/common/radio-toggle.js (hunk 1 above)
Comment (LaurieSReynolds) [reply src/__tests__/components/common/radio-toggle.js]: Oops - updated.

**Review by kevee — APPROVED** [review]
LGTM, thanks @LaurieSReynolds !

**Pull request merged**